src/container/AboutUsSection/AboutUsSection.css

```diff
@@ -0,0 +1,47 @@
+.AboutUsContent
+{
```

Comment:
To raczej kwestia stylistyki, ale my raczej piszemy nawiasy przy nazwach czyli:

.AboutUsContent {
 ...
}